deployments/helm/values.yaml

```diff
@@ -15,17 +15,53 @@ starknet:
   preset: mainnet.json
   additionalHeaders: # optional addtional headers for SN communication
 
+p2p:
+  enabled: true
+  # Set to true if node is "bootstrap"
+  bootstrap: false
+  # General config
+  config:
+    # Optional - The node self port to listen | default "10000"
+    network_tcp_port:
+    # Optional - The node data path | default "data"
+    storage_db_config_path_prefix:
+    # Optional - network.#is_none flag | default "false"
+    network_is_none:
+  # Config to include only if "bootstrap: false"
+  nodeConfig:
+    # Mandatory - The network.#is_none flag on the bootsrap server
+    network_bootstrap_peer_multiaddr_is_none:
+    # Mandatory - The bootstrap server to connect to, ip adress
+    network_bootstrap_peer_multiaddr_ip:
+    # Mandatory - The bootstrap server to connect to, port 
+    network_bootstrap_peer_multiaddr_port:
+    # Mandatory - The bootstrap server to connect to, uid
+    network_bootstrap_peer_multiaddr_uid:
```

Comment:
since all these params have similar purpose and are all related, I suggest to nest them together:
```
bootstrap_server:
  ip:
  port:
  uid: 
  ```
  
  also, we need to be consistent in the syntax formatting. either `sneak_case` or `camelCase` but not both